harvester_e2e_tests/integration/test_backup_restore.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (c) 2022 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+from harvester_e2e_tests import utils
+from harvester_api import API
+import time
+import pytest
+import math
+
+
+pytest_plugins = [
+    'harvester_e2e_tests.fixtures.backuptarget'
+]
+
+
+IMAGE_URL = 'https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img'
+IMAGE_DISPLAY_NAME = 'ubuntu-image'
+TEST_VM_NAME = 'test'
+TEST_RESTORED_VM_NAME = 'test-restored'
+
+
+@pytest.fixture(scope='class')
+def login(request):
+    endpoint = request.config.getoption('--endpoint')
+    username = request.config.getoption('--username')
+    password = request.config.getoption('--password')
+    api = API.login(endpoint, username, password, ssl_verify=False)
+    return api
+
+
+def wait_for_image_ready(api, name):
+    image = api.images.get(name)
+    while ('status' not in image[1] or
+            'progress' not in image[1]['status'] or
+            image[1]['status']['progress'] != 100):
+        image = api.images.get(name)
+        time.sleep(5)
+
+
+@pytest.mark.backupnfs
+class TestBackupRestoreNFS:
+    @pytest.fixture(scope='class')
+    def prepare(self, request, admin_session, login,
+                harvester_api_endpoints, backuptarget_nfs):
+        api = login
+
+        image = api.images.create_by_url(IMAGE_DISPLAY_NAME, IMAGE_URL)
+        assert image[0] == 201, f'Failed to create image, status code {image[0]}'

```

Comment:
suggest to use
```python
assert expected_value == value
# assert 201 == image[0]
```
instead, this will make the output more easy to read.
the output would like:
```
self = <test_vm_functions.TestVMOperations object at 0x7fef2809e860>
api_client = HarvesterAPI('https://10.84.102.32', 'Bearer token-zln9n:z2s7fwd5vxv4clzcgshwvd6fvvfslpsxkhgcszrv4kwd84pwh97g4n')
unique_vm_name = 'vm-12-21-12h34m55s241016', wait_timeout = 900

    def test_abort_migrate(self, api_client, unique_vm_name, wait_timeout):
        code, host_data = api_client.hosts.get()
        assert 200 == code, (code, host_data)
        code, data = api_client.vms.get_status(unique_vm_name)
        cur_host = data['status']['nodeName']
    
        new_host = next(h['id'] for h in host_data['data'] if cur_host != h['id'])
    
        code, data = api_client.vms.migrate(unique_vm_name, new_host)
>       assert 204 == code, (code, data)
E       AssertionError: (500, "Can't migrate the VM, the VM is not in ready status")
E       assert 204 == 500

harvester_e2e_tests/integration/test_vm_functions.py:297: AssertionError
```